kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -192,8 +230,10 @@
       CoachAppBarPage,
       PlanHeader,
       Recipients,
+      AverageScoreTooltip,
+      ReportsControls,
     },
-    mixins: [commonCoreStrings],
+    mixins: [commonCoach, commonCoreStrings],
```

Comment:
Which specific props/methods/etc are we adding the `commonCoach` mixin back in here to access?

If you look at the git blame for this line, you will see that the previous change was the removal of this mixin in this commit: https://github.com/learningequality/kolibri/commit/75eb02cf6336aed2d456eff158fde4ae9667fca5

I would like to maintain this direction of travel if possible, and can give some more targeted advice on how to include the specific things needed here, rather than reintroducing the mixin (which has such a huge array of things in it, it becomes very hard to reason about what is needed and what is not).

Reply:
className, learners, groups, getExamAvgScore, getExamStatusTally, getLearnersForExam and some components like Score and StatusSummary were used. I have added them separately using mapState and mapGetters!